code/game/objects/items/food/snacks.dm

```diff
@@ -312,3 +312,73 @@ GLOBAL_LIST_INIT(safe_peanut_types, populate_safe_peanut_types())
 	qdel(sample)
 
 	. = ..()
+
+
+/obj/item/food/cornchips
+	name = "boritos corn chips"
+	desc = "Triangular corn chips. They do seem a bit bland but would probably go well with some kind of dipping sauce."
+	icon_state = "boritos"
+	trash_type = /obj/item/trash/boritos
+	bite_consumption = 2
+	food_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/cooking_oil = 2, /datum/reagent/consumable/salt = 3)
+	junkiness = 20
+	custom_price = PAYCHECK_ASSISTANT * 0.8  //we are filled to the brim with flavor
+	tastes = list("fried corn" = 1)
+	foodtypes = JUNKFOOD | FRIED
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/cornchips/MakeLeaveTrash()
+	if(trash_type)
```

Comment:
why the check? they all have a trash type

Reply:
it was in the original code, Im too afraid to touch it but that part of the code makes it pop when stepped on